src/components/Entry/Summary/SidePanel/index.tsx

```diff
@@ -96,80 +96,87 @@ const SidePanel = ({
       {['interpro', 'pfam'].includes(
         // Only receiving new annotations for pfam and interpro
         metadata.source_database.toLowerCase()
-      ) && (
-        <div>
-          <Tooltip
-            title={
-              'You may suggest updates to the annotation of this entry using this form. Suggestions will be sent to ' +
-              'our curators for review and, if acceptable, will be included in the next public release of InterPro. It is ' +
-              'helpful if you can include literature references supporting your annotation suggestion.'
-            }
-          >
-            <DropDownButton
-              label="Add your annotation"
-              icon="&#xf303;"
-              extraClasses={css('annotation')}
+      ) &&
+        !metadata.is_removed && (
+          <div>
+            <Tooltip
+              title={
```

Comment:
The tooltip is too long, can you make it on multiple lines?

Reply:
That's a site-wide issue, reported in #518.
I may be able to fix it later this month, otherwise @gustavo-salazar will take care of it.